tests/indexes/tests.py

```diff
@@ -66,8 +69,27 @@ def test_index_together_single_list(self):
         index_sql = connection.schema_editor()._model_indexes_sql(IndexTogetherSingleList)
         self.assertEqual(len(index_sql), 1)
 
-    @skipUnless(connection.vendor == 'postgresql', "This is a postgresql-specific issue")
-    def test_postgresql_text_indexes(self):
+
+@skipIf(connection.vendor == 'postgresql', 'opclasses are PostgreSQL only')
+class SchemaIndexesNotPostgreSQLTests(TransactionTestCase):
+    available_apps = ['indexes']
+
+    def test_create_index_ignores_opclasses(self):
+        index = Index(
+            name='test_ops_class',
+            fields=['headline'],
+            opclasses=['varchar_pattern_ops'],
+        )
+
+        with connection.schema_editor() as editor:
+            editor.add_index(IndexedArticle2, index)
+
```

Comment:
Could you add a comment here that says the `add_index` would fail if we weren't ignoring the opclass.